torchvision/prototype/transforms/_auto_augment.py

```diff
@@ -1,18 +1,31 @@
 import math
-from typing import Any, Dict, Tuple, Optional, Callable, List, cast, TypeVar
+from typing import Any, Dict, Tuple, Optional, Callable, List, cast, TypeVar, Union
 
 import PIL.Image
 import torch
 from torchvision.prototype import features
 from torchvision.prototype.transforms import Transform, InterpolationMode, AutoAugmentPolicy, functional as F
-from torchvision.prototype.utils._internal import apply_recursively
+from torchvision.prototype.utils._internal import query_recursively
+from torchvision.transforms.functional import pil_to_tensor, to_pil_image
 
-from ._utils import query_image, get_image_dimensions
+from ._utils import get_image_dimensions
 
 K = TypeVar("K")
 V = TypeVar("V")
 
 
+def _put_into_sample(sample: Any, id: Tuple[Any, ...], item: Any) -> Any:
+    if not id:
+        return item
+
+    parent = sample
+    for key in id[:-1]:
+        parent = parent[key]
+
+    parent[id[-1]] = item
+    return sample
```

Comment:
Although this looks like a new concept, it was in fact copy-pasted from #5422 where we need the same kind of functionality. I suggest to leave this here for now and let #5422 handle the more elaborate case.

Reply:
Should we put it on `._utils`?

Reply:
It probably should be in `torchvision.utils._internal` since it will be needed in `torchvision.prototype.datasets`. Given that we currently only need it there, I would leave it to #5422 to do this properly and remove the definition here.